proto/api/node/node.proto

```diff
@@ -34,6 +34,24 @@ message SvidUpdate {
     repeated spire.common.RegistrationEntry registration_entries = 3;
 }
 
+// JSR is a JWT SVID signing request.
+message JSR {
+     // SPIFFE ID of the workload
+    string spiffe_id = 1;
+
+    // List of intended audience
+    repeated string audience = 2;
```

Comment:
Side note: any thoughts on where token lifetime might get configured?

Reply:
expiration isn't part of x509-SVID CSR's and the expiration is configured in one of two ways:
1) Agent SVID's get the default TTL
2) Workload SVID's get a TTL based on the registration entry.

I intended JWT-SVID's to follow suite.